translations/fr/general.yml

```diff
@@ -1,5 +1,5 @@
 goal: Objectif
-goals: Buts
+goals: Objectifs
```

Comment:
@brockfanning this looks right to me. Buts seems to mean goals in the scoring sense e.g. points.

Disclaimer: I don't speak French